lib/minitest.rb

```diff
@@ -45,6 +45,12 @@ module Minitest
   mc.send :attr_accessor, :info_signal
   self.info_signal = "INFO"
 
+  SUITE_ORDERINGS = [:sorted, :random]
+
+  def self.suite_order
+    @@suite_order || :random
```

Comment:
If you add `:order => :random` to the default `options` hash `process_args`, would it eliminate this conditional?